website/markdown/docs/commands/build.mdx

```diff
@@ -0,0 +1,49 @@
+---
+name: Build schemes
+excerpt: 'Learn how to build your Tuist projects by simply using the build command that is optimized for minimal configuration.'
+---
+
+# Build schemes
+
+Traditionally,
```

Comment:
I'd remove this part because I find it subjective to the way you use the other tools

Reply:
I'd keep it but removing the references to other tools. I think it's important to highlight where the motivation of building automation into Tuist came from.